iohk/Buildkite/Pipeline.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
I don't see PipelineDefinition nor PipelineStep used anywhere (except in tests), am I missing something? :)